src/Mvc/Mvc.Core/src/Infrastructure/ActionResultStatusCodeAttribute.cs

```diff
@@ -20,7 +20,7 @@ namespace Microsoft.AspNetCore.Mvc.Infrastructure
     /// StatusCodeResult([ActionResultStatusCodeParameter] int statusCode)
     /// </example>
     [AttributeUsage(AttributeTargets.Parameter, AllowMultiple = false, Inherited = false)]
-    internal sealed class ActionResultStatusCodeAttribute : Attribute
+    public sealed class ActionResultStatusCodeAttribute : Attribute
```

Comment:
why tho?

Reply:
We look at these attributes as part of the Api Analyzer which requires these to be part of the ref assemblies. I'm going to slap an EditorBrowseable Never on them

Reply:
I don't think that's necessary. These should probably just be public.